hrjobcontract/js/test/controllers/contract_test.js

```diff
@@ -1,22 +1,79 @@
 define([
+    'common/moment',
     'common/angularMocks',
-    'job-contract/app'
-], function () {
+    'job-contract/app',
+    'job-contract/controllers/contract'
```

Comment:
You shouldn't include this, the `job-contract/controllers/contract` should already be included in `job-contract/app`